util/timer/parseTime.js

```
@@ -1,7 +1,7 @@
 module.exports = function(time, currentUnixTime){
-	let timeInSeconds = parseFloat(time.slice(0, time.length - 1))
-	let letter = time.slice(time.length - 1)
-	if(!isNaN(letter)) return parseFloat(time) * 60;
+	let timeInSeconds = parseFloat(time)
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)